std/random.d

```diff
@@ -1349,39 +1482,17 @@ if (isUnsigned!UIntType)
     {
         return this;
     }
-
-
-    /**
-     * Compares against $(D_PARAM rhs) for equality.
-     */
-    bool opEquals(ref const XorshiftEngine rhs) const @safe pure nothrow @nogc
-    {
-        return seeds_ == rhs.seeds_;
-    }
-
-
-  private:
-    static void sanitizeSeeds(ref UIntType[size] seeds) @safe pure nothrow @nogc
```

Comment:
Deleted because it was incorporated into `seed` like in @9il's reworked version of XorshiftGenerator.